app/src/App/OnDeviceDisplayApp.tsx

```diff
@@ -252,23 +252,31 @@ export function OnDeviceDisplayAppRoutes(): JSX.Element {
   `
 
   return (
-    <Switch>
+    <Routes>
       {ON_DEVICE_DISPLAY_PATHS.map(path => (
-        <Route key={path} exact path={path}>
-          <Box css={TOUCH_SCREEN_STYLE} ref={scrollRef}>
-            <ModalPortalRoot />
-            {getPathComponent(path)}
-          </Box>
-        </Route>
+        <Route
+          key={path}
+          path={path}
+          element={
+            <Box css={TOUCH_SCREEN_STYLE} ref={scrollRef}>
+              <ModalPortalRoot />
+              {getPathComponent(path)}
+            </Box>
+          }
+        />
       ))}
-      {targetPath != null && <Redirect exact from="/" to={targetPath} />}
-    </Switch>
+      {targetPath != null && (
+        <Route path="/" element={<Navigate to={targetPath} />} />
+      )}
```

Comment:
https://stackoverflow.com/questions/69868956/how-can-i-redirect-in-react-router-v6

```suggestion
      <Route path="*" element={<Navigate to={targetPath} replace />} />
```

Reply:
thank you so much koji!